ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -387,7 +387,7 @@ const GENE_DETAIL_SECTIONS = [
     color: 'red',
     description: 'HaploInsufficient',
     label: 'HI',
-    showDetails: gene => gene.cnSensitivity.phi && gene.cnSensitivity.phi > HI_THRESHOLD,
+    showDetails: gene => gene.cnSensitivity?.phi && gene.cnSensitivity.phi > HI_THRESHOLD,
```

Comment:
We don't fetch the `cnSensitivity` field of the gene for the phenotype-prioritized gene data. So it can be undefined.

Reply:
as I said above, we need to fetch the same data as for variants, we want to show everything

Reply:
Okay